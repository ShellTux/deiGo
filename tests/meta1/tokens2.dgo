fmt.Println fmt1Println fmtlPrintln fmtOPrintln fmt+Println fmt Println
strconv.Atoi strconv2Atoi strconvgAtoi strconvEAtoi strconv-Atoi strconv Atoi
os.Args os3Args osxArgs osMArgs os,Args os Args
