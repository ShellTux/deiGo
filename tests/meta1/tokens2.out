PRINT
IDENTIFIER(fmt1Println)
IDENTIFIER(fmtlPrintln)
IDENTIFIER(fmtOPrintln)
IDENTIFIER(fmt)
PLUS
IDENTIFIER(Println)
IDENTIFIER(fmt)
IDENTIFIER(Println)
SEMICOLON
PARSEINT
IDENTIFIER(strconv2Atoi)
IDENTIFIER(strconvgAtoi)
IDENTIFIER(strconvEAtoi)
IDENTIFIER(strconv)
MINUS
IDENTIFIER(Atoi)
IDENTIFIER(strconv)
IDENTIFIER(Atoi)
SEMICOLON
CMDARGS
IDENTIFIER(os3Args)
IDENTIFIER(osxArgs)
IDENTIFIER(osMArgs)
IDENTIFIER(os)
COMMA
IDENTIFIER(Args)
IDENTIFIER(os)
IDENTIFIER(Args)
SEMICOLON
